paddle/function/ContextProjectionOp.cpp

```diff
@@ -350,13 +285,5 @@ REGISTER_TYPED_FUNC(ContextProjectionForward,
 REGISTER_TYPED_FUNC(ContextProjectionBackward,
                     GPU,
                     ContextProjectionBackwardFunc);
-#if 0
-REGISTER_TYPED_FUNC(ContextProjectionBackwardData,
```

Comment:
Just keep and update these interfaces. Do we need to have to output SequenceArg?